cluster/images/hyperkube/static-pods/etcd.json

```diff
@@ -10,7 +10,7 @@
     "containers": [
       {
         "name": "etcd",
-        "image": "gcr.io/google_containers/etcd-ARCH:2.3.7",
+        "image": "gcr.io/google_containers/etcd-ARCH:2.2.1",
```

Comment:
Umm, wait. 
This is wrong, it should be 2.2.5 (since arm64 and ppc64le only works on 2.2.5, so that version has been used there instead)

I can fix it in my other PR...